www/js/share/components/accounts/accounts.controller.js

```diff
@@ -44,12 +45,51 @@
 
     vm.updateAccount = function(_selectedAccount) {
       appStateService.isChangedAccount = true;
-      sessionStorage.removeItem('start');
+			appStateService.isCheckedAccountType = false;
+			sessionStorage.removeItem('start');
+			sessionStorage.removeItem('_interval');
       utilsService.spinnerLogo.start();
       accountService.setDefault(_selectedAccount);
       accountService.validate();
       updateSymbols();
+      $rootScope.$broadcast('changedAccount');
```

Comment:
it doesn't need to broadcast a message through $rootScope. you can use $root instead since broadcasting is global in angularjs.